example/register_device/lib/ThingooConnector/ThingooConnector.cpp

```diff
@@ -0,0 +1,81 @@
+/***************************************************************
+   esp8266Thingoo is a library for the ESP8266/Arduino platform
+   to communicate with Thingoo API
+   Built for KNI Project "Thingoo" https://github.com/ThingooKNI/
+ **************************************************************/
+
+#include "ThingooConnector.h"
+
+#include <Arduino.h>
+#include <WiFiClientSecure.h>
+#include <ESP8266HTTPClient.h>
+#include <ArduinoJson.h>
+#include <WiFiClientSecureBearSSL.h>
+
+ThingooConnector::ThingooConnector(const char* host)
+{
+
+    https_port = 443; //HTTPS= 443
+    register_endpoint = "/auth/realms/Thingoo/protocol/openid-connect/token";
+    readings = "/readings";
+    devices = "/devices";
+    _host = host;
+}
+
+void ThingooConnector::set_client_credentials(String client_id, String secret_key)
+{
+
+    _client_id = client_id;
+    _secret_key = secret_key;
+}
+
+void ThingooConnector::set_fingerprint(const char* fingerprint)
```

Comment:
This method name doesn't tell much to the user who is not familiar with how https is handled on esp8266. Maybe call it `set_ssl_certificate_sha1_fingerprint`